bot_identifier.py

```diff
@@ -300,16 +308,21 @@ def run(self, df, step=2):
     # nltk.download("wordnet")
     
     # Input File
-    df = pd.read_csv("./data/results/collected_data_one_year_microsoft_May1st.csv") # here is the raw dataset file path
+    # df = pd.read_csv("./data/results/collected_data_one_year_microsoft_May1st.csv") # here is the raw dataset file path
```

Comment:
Hi Habib,
Could you please clean these personal path part and only leave the exception handling part?

Reply:
Cleaned up path.